CHANGELOG.next.asciidoc

```diff
@@ -46,6 +46,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Added `container.image.name` to `journald` Filebeat input's Docker-specific translated fields. {pull}40450[40450]
 - Change log.file.path field in awscloudwatch input to nested object. {pull}41099[41099]
 - Remove deprecated awscloudwatch field from Filebeat. {pull}41089[41089]
+- Add support to source AWS cloudwatch logs from linked accounts. {pull}41188[41188]
```

Comment:
This is under breaking change section, we can move it to Added section.

Reply:
+1

Reply:
thank you @kaiyan-sheng , I corrected this with https://github.com/elastic/beats/pull/41188/commits/e280d23c0441465e1de27c9dd11e8e2db5eeb11a